packages/autoloader/src/AutoloadGenerator.php

```diff
@@ -288,6 +289,28 @@ private function getFileMap( array $autoloads, Filesystem $filesystem, $vendorPa
 		return 'array( ' . PHP_EOL . $fileMapString . ');' . PHP_EOL;
 	}
 
+	/**
+	 * Take the main package object and generate the package metadata file to be later used by
+	 * the autoloader to determine loading precedence.
+	 *
+	 * @param PackageInterface $mainPackage the main package object.
+	 *
+	 * @return string $metadata
+	 */
+	private function getAutoloadMeta( PackageInterface $mainPackage ) {
+
+		$version = $mainPackage->getVersion();
```

Comment:
I think `$mainPackage->getVersion()` provides the version of the consuming plugin, since that is the main package. We want the version of the `automattic/jetpack-autoloader` package, right?